features/old/applications/providers/create_application.feature

```diff
@@ -84,7 +84,8 @@ Feature: Create application
        And I fill in "Name" with "UltimateWidget"
        And I fill in "Description" with "Awesome ultimate super widget"
        And I select "second_app_plan" from "Application plan"
-      Then I should see "Create a service plan"
+      # FIXME: this is a false positive, the element is invisible
```

Comment:
@josemigallas Create a Jira issue for this plz and reference it here, because otherwise we will forget for sure